tests/testthat/test-sprintf_linter.R

```diff
@@ -0,0 +1,62 @@
+context("sprintf_linter")
+test_that("returns the correct linting", {
+  expect_lint("sprintf('hello')",
+    NULL,
+    sprintf_linter)
+
+  expect_lint("sprintf('hello', 1)",
+    NULL,
+    sprintf_linter)
+
+  expect_lint("sprintf('hello %d', 1)",
```

Comment:
IMHO this should be a warning (unused fmt argument). What do you think?

Reply:
I meant `sprintf('hello', 1)`

Reply:
https://github.com/jimhester/lintr/pull/578#issuecomment-732144405